src/quacc/recipes/onetep/_base.py

```diff
@@ -62,11 +50,9 @@ def base_fn(
 
     atoms.calc = Onetep(
         input_atoms=atoms,
-        preset=preset,
-        template=template,
-        profile=profile,
         calc_defaults=calc_defaults,
-        parallel_info=parallel_info,
+        pseudo_path=SETTINGS.ONETEP_PP_PATH,
+        profile=OnetepProfile(argv=SETTINGS.ONETEP_PARALLEL_CMD),
```

Comment:
Note to self: I think this should actually just be a strong containing the parallel command concatenate with the executable name, both of which are defined as settings now.